### PR TITLE
Remining UIType fixes from pull request #50.

### PR DESCRIPTION
**Pull request by MHumm**
Added UITypes to uses clause where suggested in pull request #50 but using the proper define.

### CODE REVIEW

File: jvcl/run/JvFullColorCtrls.pas [hunk 1]
```diff
@@ -742,6 +742,9 @@ implementation
 uses
   RTLConsts, TypInfo, Forms,
   JclMath, JclLogic, // For EnsureRange and Min/Max
+  {$IFDEF HAS_UNIT_SYSTEM_UITYPES}
+  UITypes,
```
Comment (obones) [inline jvcl/run/JvFullColorCtrls.pas]: The UnitName should really be prefixed, the compiler supports it.
So please replace `UITypes` by `System.UITypes`

File: jvcl/run/JvGIF.pas [hunk 2]
```diff
@@ -33,6 +33,9 @@ interface
   {$IFDEF UNITVERSIONING}
   JclUnitVersioning,
   {$ENDIF UNITVERSIONING}
+  {$IFDEF HAS_UNIT_SYSTEM_UITYPES}
+  UITypes,
```
Comment (obones) [inline jvcl/run/JvGIF.pas]: Same remark

File: jvcl/run/JvHLEditor.pas [hunk 3]
```diff
@@ -96,6 +96,9 @@ interface
   JclUnitVersioning,
   {$ENDIF UNITVERSIONING}
   SysUtils, Classes, Graphics,
+  {$IFDEF HAS_UNIT_SYSTEM_UITYPES}
+  UITypes,
```
Comment (obones) [inline jvcl/run/JvHLEditor.pas]: Sam remark

**Comment by MHumm** [comment]
Updated code as requested.

**Pull request merged**